src/services/account/services/accountApiService.js

```diff
@@ -76,7 +92,10 @@ const accountServiceHooks = {
 		update: [disallow('external')],
 		patch: [
 			authenticate('jwt'),
+			iff(isProvider('external'), preventPopulate),
 			sanitizeUsername,
+			validateUserName,
```

Comment:
why is this only checked on patch, but not on create?

Reply:
added to create method as well
:+1: